scss/_patterns_cta.scss

```diff
@@ -0,0 +1,13 @@
+@import 'settings';
+
+@mixin vf-p-cta {
+  .p-cta-block {
+    // same as %section-padding--shallow
+    padding-bottom: $spv--x-large;
+  }
+
+  .p-cta-block > .p-cta-block__link {
```

Comment:
It's not strictly necessary, no - I used it since the component's markup is so small, I couldn't foresee an instance where it wouldn't be an immediate descendant. Do you think it's worth making it a simple descendant selector?

Reply:
Switched from child selector to descendant selector